hardhat.config.ts

```diff
@@ -102,7 +102,7 @@ const config: HardhatUserConfig = {
   solidity: {
     compilers: [
       {
-        version: "0.8.0",
+        version: "0.8.11",
```

Comment:
is it beneficial to only have one compiler version and update all files?

Reply:
Not sure about this